tests/api/test_api.py

```diff
@@ -237,7 +237,7 @@ def test_download_file_from_url(cript_api: cript.API, tmp_path) -> None:
     # the path it will save it to will be `tmp_path/downloaded_file_name.json`
     path_to_save_file: Path = tmp_path / "downloaded_file_name"
 
-    cript_api.download_file(file_source=url_to_download_file, destination_path=str(path_to_save_file))
+    cript_api.download_file(url_to_download_file, str(path_to_save_file))
```

Comment:
I think this should work

```py
cript_api.download_file(file_source=url_to_download_file, destination_path=str(path_to_save_file))
```

I don't understand why it would break